library/database/class.database.php

```diff
@@ -150,11 +150,15 @@ public function commitTransaction() {
      */
     protected function newPDO($Dsn, $User, $Password) {
         try {
+            if (strpos($Dsn, 'charset=') === false) {
```

Comment:
I actually wanted to do that at first but I feared that changing the location of where the charset is set could cause problems.